styles.css

```diff
@@ -368,6 +367,61 @@ header {
   background: rgba(255, 255, 255, 0.24);
 }
 
+.modal::before {
+  content: '';
+  vertical-align: middle;
+}
+
+.modal {
+  background-color: rgba(60, 60, 60, 0.24);
+  position: absolute;
+  width: 100%;
+  height: 100vh;
+  left: 0;
+  z-index: 20;
+  padding: 60px 16px;
+  backdrop-filter: blur(3px);
+}
```

Comment:
- [ ] Your modal is perfect, just one thing on the mobo version, I can see the rest of the modal and can't see the two buttons. You can fix it by adding `overflow: scroll;` to this `.modal` class to make it scrollable, so you can see the whole modal.


![image](https://user-images.githubusercontent.com/35535130/162094548-e4d3821e-344b-4907-83ec-eb1a574ddfac.png)